app/gui2/src/components/GraphEditor.vue

```diff
@@ -252,8 +254,11 @@ useEvent(window, 'keydown', (event) => {
     (!keyboardBusyExceptIn(documentationEditorArea.value) && undoBindingsHandler(event)) ||
     (!keyboardBusy() && graphBindingsHandler(event)) ||
     (!keyboardBusyExceptIn(codeEditorArea.value) && codeEditorHandler(event)) ||
-    (!keyboardBusyExceptIn(documentationEditorArea.value) && documentationEditorHandler(event))
+    (!keyboardBusyExceptIn(documentationEditorArea.value) && documentationEditorHandler(event)) ||
+    (!keyboardBusy() && graphNavigator.events.keydown(event))
 })
+useEvent(window, 'keyup', (event) => graphNavigator.events.keyup(event))
```

Comment:
Imo this should be part of the `graphNavigator` itself (or event `useArrows`). It should be harmless to always handle `keyup` if all it does is ends an action that had to first be started by a handled `keydown`.